couchbase-lite-core/check_builds/pkg_data.yaml.j2

```diff
@@ -0,0 +1,66 @@
+{# Map release names to the set of expected platforms. Each entry has the form #}
+{#     platform [ : architecture ] #}
+{# If :architecture is empty, the filename will use the default arch for the #}
+{# platform's packaging style (see below). #}
+{##}
+{# !!! If you add an all-new platform here, be sure to add it to the filebits_for map as well !!! #}
+{##}
+{%
+  set archive_platforms_for = {
```

Comment:
probably use "platforms_for" instead of "archive_platforms_for" to be consistent with other products.